build

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -x
+
+# Run unit tests and calculate code coverage.
+rm -f .coverage &&
+for dir in plsync ; do
+  PYTHONPATH=$PYTHONPATH:$PWD/site-packages coverage run --append --source ./ \
```

Comment:
Good question. Actually this `PYTHONPATH` is unnecessary since the operator repo does not have a site-packages directory. So, I've removed it.